packages/widgets/src/menubar.ts

```diff
@@ -1110,14 +1114,16 @@ export namespace MenuBar {
 export interface IOverflowMenuOptions {
   /**
    * Determines if a overflow menu appears when the menu items overflow.
+   *
+   * Defaults to `true`.
    */
   overflowMenuVisible: boolean;
```

Comment:
```suggestion
  isVisible: boolean;
```

Since `IOverflowMenuOptions` already has "overflow menu" in the name, I propose using the same `isVisible` flag that is used elsewhere in the API.

Reply:
Sounds good :+1: